lib/igniter.ex

```diff
@@ -1507,68 +1495,82 @@ defmodule Igniter do
     Rewrite.Source.update(source, key, value, opts)
   end
 
-  defp display_warnings(%{warnings: []}, _title), do: :ok
-
-  defp display_warnings(%{warnings: warnings}, title) do
-    Mix.shell().info("\n#{title} - #{IO.ANSI.yellow()}Warnings:#{IO.ANSI.reset()}\n")
-
-    warnings =
-      warnings
-      |> Enum.map_join("\n\n", fn error ->
-        if is_binary(error) do
-          "* #{IO.ANSI.yellow()}#{error}#{IO.ANSI.reset()}"
-        else
-          "* #{IO.ANSI.yellow()}#{Exception.format(:error, error)}#{IO.ANSI.reset()}"
-        end
-      end)
-
-    Mix.shell().info(warnings <> "\n\n")
+  @doc false
+  def display_issues(igniter) do
+    igniter.issues
+    |> Enum.reverse()
+    |> Enum.map(fn error ->
+      ["* ", :red, format_error(error)]
+    end)
+    |> display_list([:red, "Issues:"])
   end
 
-  defp display_notices(igniter) do
-    case igniter.notices do
-      [] ->
-        :ok
-
-      notices ->
-        notices =
-          Enum.map_join(notices, "\n\n", fn notice ->
-            "#{IO.ANSI.green()}#{notice}#{IO.ANSI.reset()}"
-          end)
-
-        Mix.shell().info("\n" <> notices)
-    end
+  @doc false
+  def display_warnings(igniter, title) do
+    igniter.warnings
+    |> Enum.reverse()
+    |> Enum.map(fn error ->
+      ["* ", :yellow, format_error(error)]
```

Comment:
Do you prefer using the atoms directly over using the `IO.ANSI.<color>` functions?